projects/systelab-components/sass/modern/_agGrid.scss

```diff
@@ -2,13 +2,14 @@ $checkbox-text-gap: $slab-checkbox-height * 1.5 !default;
 
 $icon-rowaction-font-size: 20px;
 
+$header-menu-width: 39px;
```

Comment:
I would rename it to "context-menu-width" instead of header. This is the size of the context menu